### PR TITLE
Update README with spelling fix in command

### PR DESCRIPTION
**Pull request by janemiceli**

### CODE REVIEW

File: roles/gui/tasks/log.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (Blue-Dog-Archolite) [inline roles/gui/tasks/log.yml]: Remove Debug

File: roles/post-brew/tasks/pipfiles/vim3/Pipfile [hunk 2]
```diff
@@ -17,4 +17,4 @@ yapf = "*"
 black = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "4.1"
```
Comment (Blue-Dog-Archolite) [inline roles/post-brew/tasks/pipfiles/vim3/Pipfile]: This version does not exist.

File: roles/post-brew/tasks/vim-pip-installs.yml [hunk 3]
```diff
@@ -10,27 +14,55 @@
...
+#I am not sure on the best src location, as "./pipfiles/emacs/" doesn't exist
+#This may need to be developed more or changed completely
+#
+
 - name: Install Vim2 Packages
   shell: |
-    pipenv --two
-    pipenv install
-
+    python3 -m pipenv --two
```
Comment (Blue-Dog-Archolite) [inline roles/post-brew/tasks/vim-pip-installs.yml]: Whitespace

File: roles/pre-install/tasks/apt-install.yml [hunk 4]
```diff
@@ -4,9 +4,11 @@
   apt:
     upgrade: dist
     update_cache: yes
+  #when: ansible_virtualization_type != "xen"
```
Comment (Blue-Dog-Archolite) [inline roles/pre-install/tasks/apt-install.yml:7]: Remove commented lines.

File: roles/pre-install/tasks/directory-prep.yml [hunk 5]
```diff
@@ -6,3 +6,11 @@
     state: directory
     mode: 0777
     recurse: yes
+
+#- name: Create home directory for tools
```
Comment (Blue-Dog-Archolite) [inline roles/pre-install/tasks/directory-prep.yml]: Is this required? Docs suggest this is an expected function.

File: roles/workstation/tasks/apt-install.yml [hunk 6]
```diff
@@ -11,25 +11,23 @@
...
       - clamav
       - clamtk
       - ctags
-      - curl
       - fish
       - fonts-powerline
       - git
       - htop
       - mlocate
+      - net-tools
```
Comment (Blue-Dog-Archolite) [inline roles/workstation/tasks/apt-install.yml:23]: Sort the dependencies so reading and location of packages is easier.

File: roles/workstation/tasks/apt-repos.yml [hunk 7]
```diff
@@ -18,6 +18,7 @@
     state: present
     filename: node-12
     update_cache: yes
+  when: ansible_virtualization_type != "xen"
```
Comment (Blue-Dog-Archolite) [inline roles/workstation/tasks/apt-repos.yml]: move this check up to the workstation main level and not at the task level. dry and all that.

**Review by Blue-Dog-Archolite — CHANGES_REQUESTED** [review]

**Comment by janemiceli** [comment]
I hate the team update based on the comments here. I expect all the changes requested are in his and we can close this one without mergning.

**Pull request closed without merging**